Unstable with one of the latest commits

One of the latest commits causes dionaea to exit prematurely with the error message `[19062017 21:00:39] nfq nfq.c:116: error during nfq_unbind_pf() family 2`. Have any dependencies changed? Last good version is commit be43edcbecbd69931348891c5b529854501d103 which works perfectly fine.

--- a/modules/python/dionaea/sip/__init__.py
+++ b/modules/python/dionaea/sip/__init__.py
@@ -93,8 +93,11 @@ class SIPService(ServiceLoader):
 
         if len(daemons) > 0:
             global g_timer_cleanup
-            g_timer_cleanup = pyev.Timer(60.0, 60.0, g_default_loop, cleanup)
-            g_timer_cleanup.start()
+            if g_timer_cleanup is None:
+                g_timer_cleanup = pyev.Timer(60.0, 60.0, g_default_loop, cleanup)
+                g_timer_cleanup.start()
+            else:
+                logger.debug("Cleanup loop already started!")
         return daemons
 
 

